stable/fluent-bit/templates/config.yaml

```diff
@@ -70,6 +70,18 @@ data:
         Logstash_Format On
         Retry_Limit False
         Type  {{ .Values.backend.es.type }}
+{{ else if eq .Values.backend.type "splunk" }}
```

Comment:
By the looks of things, this if is terminated efter the `.Values.backend.es.*` ifs?

Reply:
Yeah, this breaks the .es.* options below. It should be moved down right above .Values.backend.type "http"

Reply:
nevermind. this review is old. it did in fact break things and we fixed it later.

Reply:
@kfox1111 Where has it been fixed? From what I can see it's still an issue on `master` and helm/charts#8166 isn't moving

Reply:
sigh. your right. we need to get some reviewers on that review asap.

Reply:
and this one was the one I was thinking merged: https://github.com/helm/charts/pull/7878

Reply:
and https://github.com/helm/charts/pull/7413 :/

Reply:
@unguiculus we have 3 different implementations of a bugfix on trunk due to lack of reviews and new pr's piling up for the same bug. how should we proceed?